trialRun.py

```diff
@@ -22,7 +23,7 @@ def get_unique_count(df: pd.DataFrame, col_name: pd.Index):
 
 # Transform Data
 invoice_to_weight = pd.DataFrame(df, columns=["Document No.", "Quantity"]).groupby("Document No.")
-
+print(invoice_to_weight)
```

Comment:
Is this print statement necessary?

Reply:
Nope, I just wanted to see where to get the invoice and quantity from👍

Reply:
Understood :)